docs/README.md

```diff
@@ -119,10 +119,12 @@ The compiled binary will be built to `./target/release/logdna-agent`.
 
 ### Options
 
-The agent accepts configuration from two sources, environment variables and a configuration YAML file. The default configuration yaml file is located at `/etc/logdna/config.yaml`. The following options are available:
+The agent accepts configuration from three different sources: environment variables, command line arguments and/or a
```

Comment:
I haven't included enough information about the command line arguments in the docs, I think the whole "--help" output belongs in a new doc "Using the agent on linux" or something like that. We have to reorganize the docs when we add support for Linux.

Reply:
Yeh, we'll need to add in the installation etc, we could move the configuration into a separate md and reference it from both KUBERNETES.md and Linux, or maybe we should have CLI in one place and ENV var based config in another...